tests/test_forms.py

```diff
@@ -0,0 +1,34 @@
+from django.core import mail
+from django.test import TestCase
+
+from wagtailcontact.forms import ContactForm
+
+
+class ContactFormTestCase(TestCase):
```

Comment:
Have you tested the `save()` method to ensure it does what it is supposed to? Same for the `get_to()` method.